airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -19,6 +19,36 @@
 Changelog
 ---------
 
+2.3.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Add IAM Role Credentials to S3ToRedshiftTransfer and RedshiftToS3Transfer (#18156)``
+* ``Adding missing &#39;replace&#39; param in docstring (#18241)``
```

Comment:
We seem to have issue with the escaping the single quote the `&#39;`  it appears 15 times in the PR

Reply:
Yeah :)